packages/treasury/src/stablecoinMachine.js

```diff
@@ -83,6 +54,11 @@ export async function start(zcf, privateArgs) {
   const governorPublic = E(zcf.getZoeService()).getPublicFacet(electionManager);
 
   const { feeMintAccess } = privateArgs;
+  const runMint = await zcf.registerFeeMint('RUN', feeMintAccess);
+  const { issuer: runIssuer, brand: runBrand } = runMint.getIssuerRecord();
+  zcf.setTestJig(() => ({
+    runIssuerRecord: runMint.getIssuerRecord(),
+  }));
```

Comment:
This `setTestJig` thing scares me every time I see it. I have convinced myself it's OK once or twice, but I don't recall the argument just now.

In any case, the RUN issuer is not sensitive.